tests/test_paper.py

```diff
@@ -0,0 +1,176 @@
+"""
+Paper-mode testing
+"""
+
+import os
+import trio
+import math
+from shutil import rmtree
+from exceptiongroup import BaseExceptionGroup
+from typing import (
+    AsyncContextManager,
+    Literal,
+)
+
+import pytest
+import tractor
+from tractor._exceptions import ContextCancelled
+from uuid import uuid4
+from functools import partial
+
+from piker.config import get_app_dir
+from piker.log import get_logger
+from piker.clearing._messages import Order
+from piker.pp import (
+    open_trade_ledger,
+    open_pps,
+)
+from piker.clearing import (
+    open_ems,
+)
+from piker.clearing._client import (
+    OrderBook,
+)
+
+from piker.clearing._messages import BrokerdPosition
+
+log = get_logger(__name__)
+
+
+@pytest.fixture(scope="session")
+def paper_cleanup():
+    """This fixture removes the temp directory
+    used for storing all config/ledger/pp data
+    created while testing once an entire test session
+    has completed.
```

Comment:
style for docs strings i'm trying to encourage is use of `'''` and a newline before the final closing `'''``; not a huge deal here but going forward.